tests/namespaces.bats

```diff
@@ -266,7 +271,7 @@ general_namespace() {
         expect_output "$mynamespace"
         ;;
       /*)
-        expect_output "$(readlink $namespace)"
+        expect_output "$(readlink $different)"
```

Comment:
This looks like a good catch to me, but how did this ever work?

Reply:
This case has not been tested.
Because of `$types` in L257, only the case where `different` is empty (`types[0]`) has been tested.